src/content/pages/sponsorship/sponsor.mdx

```diff
@@ -0,0 +1,111 @@
+      <HighlightItem>**34K** YouTube subscribers and **4M+** YouTube views</HighlightItem>
+      <HighlightItem>**25%** of participants from underrepresented groups in tech</HighlightItem>
+</HighlightCard>
+
+Join **EuroPython 2025 from 14 – 20 July** at the Prague Congress Centre (PCC) to connect, showcase your company, and shape the future of Python!
+
+<YouTube id="oW8FHL4PiuI" alt="Hear from our sponsors at EuroPython 2023" />
+<figcaption>Hear from our sponsors at EuroPython 2023</figcaption>
+
+Explore the benefits of sponsoring EuroPython:
+<BenefitsList>
+<BenefitItem icon="network" title="Brand Exposure">Boost your brand and connect with **1,500 Pythonistas** at this leading Python event.</BenefitItem>
+<BenefitItem icon="target" title="Product Promotion">Showcase your product to an engaged audience and **turn visibility into high-quality leads**.</BenefitItem>
+<BenefitItem icon="award" title="Community Support">Build **positive brand association** and gain community respect & appreciation.</BenefitItem>
+<BenefitItem icon="transfer" title="Skill Boost">Tap into the expertise of the Python community to **upskill your team**.</BenefitItem>
+<BenefitItem icon="headhunt" title="Talent Recruitment">Access **a pool of talented Python developers** looking for their next opportunity.</BenefitItem>
+  <BenefitItem icon="rocket" title="New Business">Discover new avenues and opportunities to **expand and grow your business**.</BenefitItem>
+</BenefitsList>
+
+#### Ready to Become a EuroPython 2025 Sponsor?
+<div class="text-center">
+
+<ButtonLink url="undefined">Sign Up by undefined & Get 10% Early Bird Discount!</ButtonLink>
+
+</div>
+
+
+---
+ # Sponsor Packages
+
+At EuroPython 2025, we offer **seven main packages** to ensure
+supporting the Python community is accessible for organisations of all sizes.
+
+<div class="text-left [&>*]:mr-8 [&>*]:mb-2 mb-12">
+<ButtonLink url="undefined">Full Sponsorship Opportunities (PDF)</ButtonLink>
+<ButtonLink url="undefined">Sign up now!</ButtonLink>
+</div>
+
+
+<SponsorTiers signupLink="undefined" />
+
+## Optional Add-ons
+
+In addition to our main sponsorship packages, we offer **premium optional add-ons**
+designed to amplify your sponsorship impact and visibility.
+They are available for sponsors at **Silver Package level and above**.
+Here are some popular examples to give you a taste:
+
+- **Technical talk slot (30 min)**: €5000
+- **Tutorial slot (3h)**: €5000
+- **Poster slot**: €3000
+- **Lanyards Sponsor**: €4000
+- **Childcare Sponsor**: €4000
+- **Wifi Sponsor**: 4000
+- **Hackathon Sponsor**: TBD
+- **Talk Room Sponsor**: TBD
+- **Talk Room Sponsor 1 day only**: TBD
+- **Open Space sponsor**: TBD
+- **Quiet Room sponsorship**: TBD
+- **Finaid sponsor**: TBD
+- **Recruiting Session / Talent Track**: TBD
+- **Social Event Sponsor**: TBD
+- **Speaker Dinner Sponsor**: TBD
+- **Media package addon**: €5000
+- And many more exciting options!
```

Comment:
As some add-ons have TBD values might make sense to just list all the above items without a value? Or list only the ones with a value?